pkg/kubelet/probe.go

```diff
@@ -66,10 +66,17 @@ func (kl *Kubelet) probeContainer(pod *api.Pod, status api.PodStatus, container
 	ref, ok := kl.containerRefManager.GetRef(containerID)
 	if !ok {
 		glog.Warningf("No ref for pod '%v' - '%v'", containerID, container.Name)
-	} else {
+		return live, err
+	}
+
+	if live != probe.Success {
 		kl.recorder.Eventf(ref, "unhealthy", "Liveness Probe Failed %v - %v", containerID, container.Name)
 	}
-	return ready, err
+	if ready != probe.Success {
+		kl.recorder.Eventf(ref, "unhealthy", "Readiness Probe Failed %v - %v", containerID, container.Name)
+	}
+
+	return live, nil
```

Comment:
At this point we know that live is probe.Success - please change live to probe.Success to make it more obvious to the reader.